tests/testthat/test-estimate_ite_aipw.R

```diff
@@ -20,8 +20,8 @@ test_that("AIPW ITE Estimated Correctly", {
   expect_true(class(ite_result) == "numeric")
 
   # Reproducible results
-  expect_equal(ite_result[1], 0.5557903, tolerance = 0.00001)
-  expect_equal(ite_result[11], 0.6201689, tolerance = 0.00001)
-  expect_equal(ite_result[91], 0.3326752, tolerance = 0.00001)
+  expect_equal(ite_result[1], 0.5578771, tolerance = 0.00001)
```

Comment:
Why do the results change?

Reply:
since the I modified `generate_cre_dataset`